app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
I think you should start separating scss files based on domain. You shouldn't put all styles in one file.

Reply:
But devise doesn't have any styleshhets. Thats why I put it on application. Should I divide it by controllers names?